python/cuml/metrics/__init__.py

```diff
@@ -34,3 +34,4 @@
 from cuml.metrics.cluster.entropy import cython_entropy as entropy
 from cuml.metrics.pairwise_distances import pairwise_distances, \
     PAIRWISE_DISTANCE_METRICS
+from cuml.metrics.hinge_loss import cython_hinge_loss as hinge_loss
```

Comment:
I wouldn't rename this function here. You should define the function as `hinge_loss` and import it without renaming like the other metrics.